functions/API/events.ts

```diff
@@ -0,0 +1,90 @@
+import {db} from '../Util/admin'
+import {Request, Response} from 'express'
+
+interface Event {
+    title: string,
+    username: string,
+    body: string,
+    invitations: [string],
+    time: Date,
+    createdAt: Date
+}
```

Comment:
Ideally it's good to put all your interfaces in a `index.d.ts` file although this is fine too.